chromebook-setup.sh

```diff
@@ -767,6 +775,9 @@ cmd_setup_fedora_kernel()
     kernel_version="$(ls vmlinuz-* | sed -e 's/vmlinuz-//')"
 
     # Generate modules.dep and map files, so modules autoload on first boot
+    if [ -z "$ROOTFS_DIR" ]; then
```

Comment:
This happens when running `chromebook-setup.sh setup_fedora_kernel ...`. It may take several repetitions of this before having the environment set up correctly to make it pass and running the complete procedure (`do_everything` or `deploy_fedora`) each time is slow due to the image copying to the target device etc.

Reply:
Got it, thanks for the explanation.